onnxruntime/python/tools/tensorrt/perf/benchmark_wrapper.py

```diff
@@ -8,7 +8,7 @@
 import re
 import pprint
 from benchmark import *
-from perf_utils import get_latest_commit_hash
+from perf_utils import *
```

Comment:
it's less preferred to import * since it pollutes the namespace. 
I typically use the from x import y pattern to import a specific set of resources.
if you are adding a reference to some other_new_func , it's better to be explicit and
`from perf_utils import get_latest_commit_hash, other_new_func`
if you really want to import all resources from perf_utils, I think it's still better to do
`import perf_utils   (and then call perf_utils.get_latest_commit_hash)`